build_container/build_container_windows.ps1

```diff
@@ -98,7 +98,10 @@ echo @"
 }
 "@ > $env:TEMP\vs_buildtools_config
 RunAndCheckError "cmd.exe" @("/s", "/c", "$env:TEMP\vs_buildtools.exe --addProductLang en-US --quiet --wait --norestart --nocache --config $env:TEMP\vs_buildtools_config")
-AddToPath (Resolve-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\*\bin\Hostx64\x64").Path
+$msvcBasePath = "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC"
+$msvcFullPath = Join-Path -Path $msvcBasePath -ChildPath "Tools\MSVC\*\bin\Hostx64\x64" -Resolve
+AddToPath $msvcFullPath
+[System.Environment]::SetEnvironmentVariable('BAZEL_VC', $msvcBasePath)
```

Comment:
`System.Environment` is the C# namespace. Do we have the right permissions to set system variables? If yes we could do `[Environment]::SetEnvironmentVariable("Path", $env:Path, [System.EnvironmentVariableTarget]::Machine)`

Reply:
this script should be run as `ContainerAdministrator` since we don't override the user here: https://github.com/envoyproxy/envoy-build-tools/blob/b1a8b205f13ce52143bcb7283c6dd0a53bcd1c6c/build_container/Dockerfile-windows2019, presumably that user should be privileged enough?